lib/suspenders/generators/app_generator.rb

```diff
@@ -181,9 +181,9 @@ def create_github_repo
       end
     end
 
-    def setup_segment_io
-      say 'Setting up Segment.io'
-      build :setup_segment_io
+    def setup_segment
+      say 'Setting up Segment'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.